modules/services/volnoti.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.volnoti;
+
+in {
+  meta.maintainers = [ maintainers.imalison ];
+
+  options = {
+    services.volnoti = { enable = mkEnableOption "Volnoti volume HUD daemon"; };
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ pkgs.volnoti ];
```

Comment:
I forgot to mention that we should make the package configurable.

Here's an example: https://github.com/nix-community/home-manager/blob/d2ade52d9616706f785f7b54956f814e2fd2ac9d/modules/programs/vivid.nix#L14-L19